packages/steveo/test/integration/kafka_test.ts

```diff
@@ -68,7 +68,7 @@ describe('Kafka Integration Test', () => {
         const randomTask = tasks[Math.floor(Math.random() * tasks.length)];
         await steveo.publish(`steveo_integration_${randomTask}`, {});
       },
-      { concurrency: 50 }
+      {concurrency: 50}
```

Comment:
I think there is some linting conflict 😅

```suggestion
      { concurrency: 50 }
```